runtime/runtime-core/common/src/aws/smithy/kotlin/runtime/businessmetrics/BusinessMetricsUtils.kt

```diff
@@ -6,14 +6,15 @@ package aws.smithy.kotlin.runtime.businessmetrics
 
 import aws.smithy.kotlin.runtime.InternalApi
 import aws.smithy.kotlin.runtime.collections.AttributeKey
+import aws.smithy.kotlin.runtime.collections.MutableAttributes
 import aws.smithy.kotlin.runtime.collections.get
 import aws.smithy.kotlin.runtime.operation.ExecutionContext
 
 /**
  * Keeps track of all business metrics along an operations execution
  */
 @InternalApi
-public val BusinessMetrics: AttributeKey<MutableSet<String>> = AttributeKey("aws.sdk.kotlin#BusinessMetrics")
+public val BusinessMetrics: AttributeKey<MutableSet<BusinessMetric>> = AttributeKey("aws.sdk.kotlin#BusinessMetrics")
```

Comment:
correctness: This is not a part of your PR but an earlier miss, we should not be referencing `aws.sdk.kotlin` in smithy-kotlin if possible

Reply:
Good catch, I'll change this to `smithy.kotlin#BusinessMetrics`